### ISSUE TITLE
Option to Not notify users of update

### ISSUE THREAD

**Issue by code3z** [NodeBB/NodeBB#9536]
Users may be notified when I update my post if I tagged them or for other reasons. There should be an option to not notify about certain updates.
https://forum.vivaldi.net/topic/60702/forum-has-been-updated/111?_=1620419466125

**Comment by julianlam** [comment]
This looks to be related to editing of posts, is that correct?

**Comment by code3z** [comment]
Yep, if I edit the post, I should have an option to not notify everyone of my edit.

**Comment by palmoni5** [comment]
I have a plugin for this.
https://github.com/palmoni5/nodebb-plugin-silent-edit

**Comment by mySebbe** [comment]
The core behavior is unchanged, but the requested workflow is now available through the maintained `nodebb-plugin-silent-edit`: <https://github.com/palmoni5/nodebb-plugin-silent-edit>. It uses the edit hooks instead of patching core, and #13906 was already closed back to this issue. I suggest closing this as provided by a plugin. A separate core proposal would need to define a public “minor/silent edit” API flag, who may use it, and whether mentions/watch notifications are all suppressed; that is a broader contract than the original report.
